src/Makefile.am

```diff
@@ -229,11 +224,10 @@ BITCOIN_CORE_H = \
   zmq/zmqabstractnotifier.h \
   zmq/zmqconfig.h\
   zmq/zmqnotificationinterface.h \
-  zmq/zmqpublishnotifier.h
-
-if ENABLE_ADDRESS_INDEXING
-  BITCOIN_CORE_H += $(ADDR_INDEX_H)
-endif
+  zmq/zmqpublishnotifier.h \
+  addressindex.h \
+  spentindex.h \
+  timestampindex.h
```

Comment:
Just a minor thing here, but I'd suggest to keep the alphabetical order for this list